### PR TITLE
Aut 1358/update problem creating login form

### PR DESCRIPTION
**Pull request by BeckaL**
## What?

Updates the "A problem with account creation / Another problem with a phone number" screen.

Screenshots:

| Locale | Before | After |
|--|--|--|
| en |Screenshot 2023-11-03 at 17 00 51|Screenshot 2023-11-02 at 16 36 39|
| cy |Screenshot 2023-11-03 at 17 04 13|Screenshot 2023-11-02 at 16 42 49|

Error messages:
| Scenario | English | Welsh |
|---|---|---|
| No information filled in |Screenshot 2023-11-03 at 16 51 51|Screenshot 2023-11-03 at 16 53 13|
| fields too long |Screenshot 2023-11-03 at 16 58 42|Screenshot 2023-11-03 at 16 59 54|

## Why?

This page was previously for users who had a problem with a non UK number. Users can now create a GOV.UK One Login with a non-UK mobile number, so the lack of a UK number should not, in itself, be a problem. However, users are still selecting this option, so users could still be experiencing problems that are not covered by the other radio buttons. This change updates the form to gather more context on this.

## Change have been demonstrated

Changes to the user interface or content should be demonstrated to Content Design and Interaction Design before being merged. This is to ensure they can make any necessary changes to Figma.
- [x] Changes to the user interface have been demonstrated

Delete this section if the PR does not change the UI.

## Performance Analysis have been informed of the change

- [x] Performance Analysis have been informed of the change

## Related PRs

[Previous change](https://github.com/govuk-one-login/authentication-frontend/pull/1176) that updated the text on the radio button and the title of the page

### CODE REVIEW

File: src/components/contact-us/contact-us-questions-validation.ts [hunk 1]
```diff
@@ -117,15 +121,6 @@ export function validateContactUsQuestionsRequest(): ValidationChainFunc {
           { value, lng: setLanguageToReflectSupportForWelsh(req.i18n.lng) }
         );
       }),
-    body("moreDetailDescription")
```
Comment (gtvj) [inline src/components/contact-us/contact-us-questions-validation.ts:120]: Hi @BeckaL, do we not need this `isLength` max length validation any more? I think `moreDetailDescription` is still present on other forms.

File: src/locales/cy/translation.json [hunk 2]
```diff
@@ -1826,8 +1826,22 @@
...
+        "section2": {
+          "header": "Beth ddigwyddodd?",
+          "hintText": "Er enghraifft ni dderbyniwyd eich rhif ffôn, neu fe gawsoch drafferth rhoi cod gwlad i mewn",
+          "errorMessage": "Rhowch beth ddigwyddodd"
+        },
+        "section3": {
+          "header": "O ba wlad mae eich rhif ffôn?",
+          "hintText": "Peidiwch â rhoi eich rhif ffôn",
+          "ifBlankErrorMessage": "Rhowch o ba wlad y mae eich rhif ffôn yn dod",
+          "ifTooLongErrorMessage": "Rhowch y wlad gan ddefnyddio llai na  256 nod"
```
Comment (gtvj) [inline src/locales/cy/translation.json:1841]: Hi @BeckaL, just spotted that there's a double space in this translation string.

**Comment by gtvj** [comment]
Hi @BeckaL - I've left a couple of comments / questions but have submitted the forms and the fields seem to be coming through OK to both Zendesk and SmartAgent (I opted not to include my email address / name when submitting)

### Zendesk submission
Screenshot 2023-11-13 at 14 28 27

### SmartAgent submission
Screenshot 2023-11-13 at 14 26 24

File: src/locales/cy/translation.json (hunk 2 above)
Comment (BeckaL) [reply src/locales/cy/translation.json]: 👍 removed

File: src/components/contact-us/contact-us-questions-validation.ts (hunk 1 above)
Comment (BeckaL) [reply src/components/contact-us/contact-us-questions-validation.ts]: An excellent spot! Think this got left in from an earlier reworking that I decided not to do, have added this back in in the relevant commit now so this shouldn't show up in the diff anywhere anymore.

**Review by gtvj — APPROVED** [review]

**Pull request merged**